graph/iid/IID.java

```diff
@@ -24,7 +24,6 @@
 
     String readableString; // for debugging
     final ByteArray bytes;
-    private int hash = 0;
```

Comment:
the hashcode is already cached in the `ByteArray`, so we can save some memory by not double caching it here